group.h

```diff
@@ -0,0 +1,26 @@
+
+class Group
+{
+public:
+    Group()
+    {
+    }
+
+    float sort(float arr[],int count)
```

Comment:
'float sort(float arr[],int count)'
do not clear what this method for

Reply:
Okay,this copied from inet =)